pkgs/applications/misc/cipher/default.nix

```diff
@@ -42,10 +41,8 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch = ''
-  	substituteInPlace data/com.github.arshubham.cipher.desktop.in \
-  		--replace xdg-open ${xdg_utils}/bin/xdg-open
```

Comment:
I still think it's a good idea to do this with `gio`.

```
substituteInPlace data/com.github.arshubham.cipher.desktop.in \
  		--replace "gio" "${glib.bin}/bin/gio"

Reply:
Hum, I thought the PR was to get rid of the substitution, but it was to remove a dependency, my bad.